BaseLib/FileTools.cpp

```diff
@@ -133,4 +133,18 @@ std::string extractPath(std::string const& pathname)
 	const std::size_t pos = findLastPathSeparator(pathname);
 	return pathname.substr(0, pos + 1);
 }
+const char pathSeparator =
+#ifdef _WIN32
+                            '\\';
+#else
+                            '/';
+#endif
+
+std::string appendPathSeparator(std::string const& path)
+{
+	if(findLastPathSeparator(path) == path.length() - 1)
```

Comment:
If `path` is empty, the check will true, as far as i can see. That's good, but not obvious.
In general I'd prefer a method like `join(path1, path2)` to this one, because that would make sure that there are (back)slashes inserted appropriately whenever you use it.
If you don't agree, please at least document the behaviour for empty strings. In particular since that is the reason why OGS does not write to `/` when the new `-o` option is omitted.